textos-estudo/react-context-api.md

```diff
@@ -0,0 +1,36 @@
+### Context API: facilitando o acesso a informações entre componentes
+Se você acompanhou os outros textos que fiz durante a mentoria, já deve ter se deparado com os props e a forma como eles se comportam entre componentes pais e filhos no React. Sabemos que  se quisermos que algum dado do componente pai persista no filho, precisamos informar esse dado na props do filho. Em alguns casos, esse processo é bem simples, passando props de um componente A para um B por exemplo, mas e se tivermos mais componentes? E se eu quiser persistir uma informação de A em D? Precisaríamos manualmente passar as props entre todos os níveis.
```

Comment:
Boa! Gostei bastante da explicação da necessidade. Podemos chamar o problema de passar os props do componente A para o D de "prop drilling". Tem um artigo de um amigo meu (sobre redux, mas que super encaixa) que pode ser útil para entender: https://samuel-santosgdmg.medium.com/the-problem-of-a-react-application-without-redux-4735cf60ac9

Reply:
Alteração feita